packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -3402,6 +3402,15 @@ export class StripeHelper extends StripeHelperBase {
           {}
       : {};
   }
+
+  async removeFirestoreCustomer(uid: string) {
+    try {
+      return await this.stripeFirestore.removeCustomerRecursive(uid);
+    } catch (error) {
+      Sentry.captureException(error);
+      return [];
+    }
```

Comment:
Currently handling the error here and returning an empty array. I'm not sure if this is the right approach

Reply:
Depends on the error, we want to throw the error up if its something we can probably retry successfully (e.g. network interruption, those bizarre one-off Firestore errors, etc).

Reply:
Retries seem to be handled by the `firestore.bulkWriter`. Currently max retries are defaulted to 5 in `StripeFirestoreBase`. (Hard limit set by Firestore is 10)

Reply:
Right, and if the retries fail, we probably want to look into it, rather than considering it 'done', and ignoring cleanup. e.g. a 10 minute outage of Firestore shouldn't allow us to proceed with deletes when we still haven't deleted them.

Reply:
Ah right, that makes sense. Let me work on getting another PR up.